src/tools/vtpipeterm/VtConsole.hpp

```diff
@@ -57,9 +57,9 @@ class VtConsole
     PROCESS_INFORMATION _piPty;
     PROCESS_INFORMATION _piClient;
 
-    HANDLE _outPipe;
-    HANDLE _inPipe;
-    HANDLE _signalPipe;
+    HANDLE _outPipe = nullptr;
+    HANDLE _inPipe = nullptr;
+    HANDLE _signalPipe = nullptr;

```

Comment:
Whoops, these should stay INVALID_HANDLE_VALUE per the discussion of pipe handles elsewhere.